pkg/webhook/controlplane/ensurer.go

```diff
@@ -186,6 +189,9 @@ func ensureKubeAPIServerCommandLineArgs(c *corev1.Container, k8sVersion *semver.
 		c.Command = extensionswebhook.EnsureStringWithPrefixContains(c.Command, "--disable-admission-plugins=",
 			"PersistentVolumeLabel", ",")
 	}
+
+	c.Args = extensionswebhook.EnsureNoStringWithPrefix(c.Args, "--service-cluster-ip-range=")
+	c.Args = append(c.Args, "--service-cluster-ip-range="+normalizeServicesIPRanges(servicesIPRanges))
```

Comment:
If we write the normailzed ranges in the status, we don't need  to change the arg here. It should be configured  correctly in g/g.

Reply:
Gotcha!

Reply:
done